src/tools/pkcs15-init.c

```diff
@@ -2330,7 +2341,8 @@ do_read_pkcs12_private_key(const char *filename, const char *passphrase,
 	*key = user_key;
 	return ncerts;
 
-error:	ossl_print_errors();
+error:
+	ossl_print_errors();
```

Comment:
should we convert this to `sc_log_openssl()` and remove the custom `ossl_print_errors()` altogether?

Reply:
Yes, updated with `sc_log_openssl`